### ISSUE TITLE
How to reproduce the results shown in the pictures?

### ISSUE THREAD

**Issue by xqroychen** [RedBlight/RaytrAMP#2]
The code can run, but not sure if it is correct.
For example the audir8, I can not obtain the same results. 
I guess that my obsX, obsY, obsZ, polX, polY, polZ setup is wrong, but not sure how to make it right.
Also how is this related to the VV,VH,HH,HV polarization.
Any help is appreciated.

**Comment by songkeke** [comment]
same problem as mine

**Comment by xiongyuup** [comment]
Excuse me, has this problem been solved

**Comment by pingpongballz** [comment]
Try with 'audir8.unv'.

**Comment by xiongyuup** [comment]
Brother, if I guarantee that the medium is PEC medium, but its permittivity is different, can I modify it?

**Comment by pingpongballz** [comment]
Yes, if the surface is PEC, you can use it as it is. The permittivity, imo is irrelevant, as the only induced currents in a PEC are on the surface.

**Comment by xiongyuup** [comment]
Thank brother, the sea surface ship RCS simulation can not use this program, brother has what open source program can calculate the sea surface ship RCS ah?

**Comment by pingpongballz** [comment]
Not much that I know of, I've been using Ansys HFSS and FEKO for my simulations

**Comment by xiongyuup** [comment]
Thanks very much.
